integration/spark/pom.xml

```diff
@@ -153,6 +153,28 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.databricks</groupId>
+      <artifactId>spark-avro_${scala.binary.version}</artifactId>
+      <version>4.0.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.avro</groupId>
+          <artifactId>avro</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-avro_${scala.binary.version}</artifactId>
```

Comment:
why can't spark2.3 and 2.4 both use databricks spark-avro ? I can understand that other way around is not possible (for both to use  spark avro)

Reply:
since we are integrated with spark, spark-avro is preferable, but as we know spark-2.3 does not have spark-avro, so anyway once in future we remove support for spark-2.3, so databricks spark-avro will be removed for our project. So here i'm avoiding out of spark project dependency as its possible in spark-2.4